src/test/workerRunner.ts

```diff
@@ -89,11 +89,11 @@ export class WorkerRunner extends EventEmitter {
     return this._project.config.timeout ? monotonicTime() + this._project.config.timeout : undefined;
   }
 
-  private _loadIfNeeded() {
+  private async _loadIfNeeded() {
```

Comment:
I think this method is used in two places, perhaps you need a rebase?

Reply:
Yep there was one in cleanup. done.